cluster/k8s-1.10.4/BUILD.bazel

```diff
@@ -7,7 +7,7 @@ load(
 container_bundle(
     name = "noderecovery_images",
     images = {
-        "localhost:33000/noderecovery:devel": "//cmd/noderecovery:noderecovery_image",
+        "localhost:33000/kubevirt/noderecovery:devel": "//cmd/noderecovery:noderecovery_image",
```

Comment:
what is supposed to be listening on 33000?

Reply:
it is port of our internal registry that available from local machine `localhost:33000->registry:5000`
so for example if you want to push and use image via internal registry you need:
- tag image to `localhost:33000/organization/image_name:image_tag`
- `docker push localhost:33000/organization/image_name:image_tag`
- use this image under our cluster via `registry:5000/organization/image_name:image_tag`

Be aware that k8s and os clusters have different registry port
- k8s `33000`
- os `33001`

It makes possible to run both clusters in the same time.